conda_mirror/conda_mirror.py

```diff
@@ -79,6 +84,7 @@ def _match(all_packages, key_glob_dict):
     matched : dict
         Iterable of package metadata dicts which match the `target_packages`
         (key, glob_value) tuples
+
```

Comment:
I don't have really any preference (space or no space) and was mostly just curious. Thanks for the clarificaiton